packages/ctt/src/copilot-instruction.ts

```diff
@@ -0,0 +1,132 @@
+import { IConstruct } from 'constructs';
+import { IResolver } from 'projen';
+
+import { ManagedTextFile, MangedTextFileOptions } from './managed-text-file';
+
+export interface CopilotInstructionOptions extends MangedTextFileOptions {
+  /**
+   * What this instruction applies to (e.g., `**\/*.ts,**\/*.tsx`)
+   */
+  readonly applyTo: string;
+
+  /**
+   * The content of the instruction
+   */
+  readonly content?: string;
+
+  /**
+   * The name of the instruction (e.g., 'typescript', 'testing')
+   */
+  readonly name: string;
+}
+
+export class CopilotInstruction extends ManagedTextFile {
+  public readonly applyTo: string;
+
+  public readonly name: string;
+
+  private contentLines: readonly string[] = [];
+
+  constructor(
+    project: IConstruct,
+    filePath: string,
```

Comment:
Make file path optional and passed in the options block. 

It should default to .github/instructions/${name}.instructions.md